packages/expo-application/src/Application.ts

```diff
@@ -53,6 +53,21 @@ export const applicationId: string | null = ExpoApplication
   ? ExpoApplication.applicationId || null
   : null;
 
+/**
+ * Indicates whether the app is using the liquid glass design. The value will be `true` when the
+ * liquid glass components are available in the app.
+ *
+ * This only checks for component availability. The value may also be `true` if the user has enabled
+ * accessibility settings that limit the liquid glass effect.
+ * To check if the user has disabled the liquid glass effect via accessibility settings, use
+ * [`AccessibilityInfo.isReduceTransparencyEnabled()`](https://reactnative.dev/docs/accessibilityinfo#isreducetransparencyenabled-ios).
```

Comment:
```suggestion
 * Indicates whether the app is using the Liquid Glass design. The value will be `true` when the
 * Liquid Glass components are available in the app.
 *
 * This only checks for component availability. The value may also be `true` if the user has enabled
 * accessibility settings that limit the Liquid Glass effect.
 * To check if the user has disabled the Liquid Glass effect via accessibility settings, use
 * [`AccessibilityInfo.isReduceTransparencyEnabled()`](https://reactnative.dev/docs/accessibilityinfo#isreducetransparencyenabled-ios).
```

Let's follow the same convention that Apple uses in its docs to reference the term: https://developer.apple.com/documentation/TechnologyOverviews/liquid-glass